config/application.rb

```diff
@@ -41,5 +41,6 @@ class Application < Rails::Application
     config.assets.precompile += %w[application.js]
     config.middleware.use ActionDispatch::Cookies
     config.middleware.use ActionDispatch::Session::CookieStore
+    config.time_zone = "Auckland"
```

Comment:
This is set on staging and prod already, as 'Pacific/Auckland'
I don't think "Auckland" is a valid one.

How about calling `.localtime` instead for now, so we don't need to retest all the other displays of date/time in the app for this change.